oap-server/server-storage-plugin/storage-shardingsphere-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/shardingsphere/PreciseDatabaseShardingAlgorithm.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.skywalking.oap.server.storage.plugin.shardingsphere;
+
+import org.apache.shardingsphere.api.sharding.standard.PreciseShardingAlgorithm;
+import org.apache.shardingsphere.api.sharding.standard.PreciseShardingValue;
+
+import java.util.Collection;
+
+/**
+ * Precise sharding algorithm to shard database.
```

Comment:
What does this sentence mean? `Precise` is an adj.. Could you re-organize?

Reply:
My guess, the actual meaning is, `Database Location Algorithm, locate the specific database instance in the given cluster`

Same questions for all `*Algorithm`.

Reply:
Precise sharding algorithm is a special item of Sharding Sphere. You can learn it more [Here](https://shardingsphere.apache.org/document/current/en/features/sharding/concept/sharding/). If necessary, i can add this description link to comment.